oaknational/Oak-Web-Application
# Feat/add unit count to units page

## Description

add units count number next to "units"

## Issue(s)

Fixes #1193 

## How to test

1. Go to unit listing page
2. Click on _______
3. You should see total unit count

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
